### PR TITLE
feat: Add headless option to nuxtIcon config

### LINKED ISSUE #24: Remove default styles and make it headless
I see that there are a few default styles added in this, which actually make the icons behave differently than it should. 
https://github.com/nuxt-modules/icon/blob/4ecfd3a4672505799aa040176da9bf737f6b5f9a/src/runtime/Icon.vue#LL56-L61C2

```html
<style scoped>
.icon {
  display: inline-block;
  vertical-align: middle;
}
</style>
```

### PR DESCRIPTION
**Pull request by fayazara**
Adds a headless option to remove `.icon` class from the icon component.

Resolves #24

### CODE REVIEW

File: src/runtime/Icon.vue [hunk 1]
```diff
@@ -30,6 +30,10 @@ const sSize = computed(() => {
   return size
 })
 
+const isHeadless = computed(() => {
+  return nuxtIcon?.headless || false
+})
+
```
Comment (manniL) [inline src/runtime/Icon.vue:36]: nit

```suggestion
const isHeadless = nuxtIcon?.headless ?? false
```

**Review by manniL — APPROVED** [review]
LGTM besides the tiny code style thing I've highlighted

**Comment by Atinux** [comment]
Looks good thank!

Can you update the readme as well to include this option?

**Comment by Atinux** [comment]
Since #38 is merged, you can set `nuxtIcon.class: ''` to have the headless option. Updated your PR and merging so you count as contributors ❤️

**Pull request merged**